extensions/amp-3q-player/validator-amp-3q-player.protoascii

```diff
@@ -36,6 +36,26 @@ tags: {  # <amp-3q-player>
     name: "data-id"
     mandatory: true
   }
+  attrs: {
```

Comment:
These are read as `data-` params, which don't need validation rules. Please undo this change.

Reply:
done